common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_backhaul.yaml

```diff
@@ -58,6 +58,10 @@ cACTION_BACKHAUL_ENABLE:
   wireless_iface_type: uint8_t  
   mem_only_psk: uint8_t
   backhaul_preferred_radio_band: uint8_t
+  ruid: sMacAddr
+  supported_channels_list:
+    _type: beerocks::message::sWifiChannel 
+    _length: [ "beerocks::message::SUPPORTED_CHANNELS_LENGTH" ]
```

Comment:
I created a fixed-length size list since I saw that in the other YAML files this supported_channels is defined as a fixed-length size list (e.g. beerocks_message_apmanager.yaml, beerocks_message_common.yaml ).
 I agree with you that it makes more sense to make it a dynamic length list, but shouldn't we do it in the rest of the place we declare this list?

Reply:
Looks like `supported_channels` is treated as a fixed-length list all over the code. IMO it should not be defined as a dynamic length list.